modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -699,9 +841,178 @@ else if (!StringUtil.equals(
 		return true;
 	}
 
+	private String _insertMethodAlphabetically(
+		String fileContent, String methodToInsert) {
+
+		int classBodyStartIndex = -1;
```

Comment:
can we simplify the names of `classBodyStartIndex`, `classBodyEndIndex` and `methodToInsert`?